### ISSUE TITLE
Easier to read branch summary on status view

### ISSUE THREAD

**Issue by yyyc514** [divmain/GitSavvy#131]
Now:

```
  BRANCH:  On branch `rendering_pipeline` tracking `origin/rendering_pipeline`. You're ahead by 5 and behind by 5.
  ROOT:    /Users/jgoebel/Library/Application Support/Sublime Text 3/Packages/GitSavvy
  HEAD:    35bd6be add guard and break

  Your working directory is clean.
```

Proposed:

```
  BRANCH:  On branch `rendering_pipeline` tracking `origin/rendering_pipeline`. 
           You're ahead by 5 and behind by 5.
  ROOT:    /Users/jgoebel/Library/Application Support/Sublime Text 3/Packages/GitSavvy
  HEAD:    35bd6be add guard and break

  Your working directory is clean.
```

This line starts to get really long.  Could we put the extended stuff on another line?

**Comment by NicoSantangelo** [comment]
I like this idea!

Even if don't need to scroll horizontally to see the line, it can be a lot of information. Looking at the code, adding newlines in [active_branch.py#L80](https://github.com/divmain/GitSavvy/blob/master/core/git_mixins/active_branch.py#L80) shouldn't break anything (I could be very wrong :P)

**Comment by yyyc514** [comment]
I'm not sure that's how I'd go about it, but I was trying to get buy-in for the idea itself before I started poking around at the code.

**Comment by divmain** [comment]
Seems like a reasonable idea :+1:.  It doesn't look like the syntax highlighting will have to be changed, which is very convenient.  Because we don't want an empty line when there is no tracking information, a new-line plus whitespace should be added in these places if the additional status information is available:

- https://github.com/divmain/GitSavvy/blob/4f359da52b9f528794039a998d53344ddf864602/core/interfaces/status.py#L133-L133
- https://github.com/divmain/GitSavvy/blob/4f359da52b9f528794039a998d53344ddf864602/core/commands/tags.py#L109-L109
- https://github.com/divmain/GitSavvy/blob/4f359da52b9f528794039a998d53344ddf864602/core/interfaces/branch.py#L76-L76

And the following should return a tuple of the two components `("On branch...", "You're ahead...")` instead of a concatenated string.

https://github.com/divmain/GitSavvy/blob/8b60d927de0bdcda58e4f75368883fa2d628afed/core/git_mixins/active_branch.py#L88-L88

**Comment by yyyc514** [comment]
#146

**Comment by divmain** [comment]
Closed with merge of #146.
